src/super_gradients/training/datasets/detection_datasets/pascal_voc_detection.py

```diff
@@ -71,17 +71,17 @@ def _load_annotation(self, sample_id: int) -> dict:
         with open(target_path, 'r') as targets_file:
             target = np.array([x.split() for x in targets_file.read().splitlines()], dtype=np.float32)
 
-        width, height = get_image_size_from_path(img_path)
+        height, width = get_image_size_from_path(img_path)
 
         # We have to rescale the targets because the images will be rescaled.
         r = min(self.input_dim[1] / height, self.input_dim[0] / width)
         target[:, :4] *= r
 
-        initial_img_shape = (width, height)
-        resized_img_shape = (int(width * r), int(height * r))
+        initial_img_shape = (height, width)
+        rescaled_img_shape = (int(height * r), int(width * r))
 
         return {"img_path": img_path, "target": target,
-                "initial_img_shape": initial_img_shape, "resized_img_shape": resized_img_shape}
+                "initial_img_shape": initial_img_shape, "rescaled_img_shape": rescaled_img_shape}
```

Comment:
Do we use at all "initial_img_shape" ?

Reply:
Not in the current code, I can remove it

Reply:
removed in 1205d4148559afc71221c13d55e4c940ec4a1c54